README.md

```diff
@@ -1 +1,56 @@
-# angular-scope-rootscope-readme
+# $scope and $rootScope
+
+## Overview
+
+## Objectives
+
+- Describe what a $scope is
+- Describe what $rootScope is
+- Describe how Angular utilises Prototypal Inheritance
+- Create a $scope variable
+- Bind the $scope variable to the View
+
+## $scope
+
+You might've noticed the magic of `$scope` in the last lesson - an object that we can assign values to and they appear in our view.
+
+Angular introduces scopes into our application. Whenever we have a new instance of a controller, that controller has a `$scope`. Everything inside our controller can access this scope, however anything outside cannot. `$scope`s can access their parent scope.
+
+`$scope` isn't just an object that can assign values to the view - it's also an extremely powerful API that allows us to do things, such as communicating between scopes, keep track of data state, watching for changes and much more - but we'll touch on this later.
```

Comment:
move this line to the last section of the README - it's a nice cliffhanger for the future READMEs